docs/guides/migration-of-security-group-rules.md

```diff
@@ -6,6 +6,8 @@ description: |-
 
 # Migrating from security_group_rules to security_group
 
+**This migration guide applies to Exoscale Terraform Provider version 0.31.3 or above.**
```

Comment:
Terraform docs support card boxes (callouts):
https://www.terraform.io/registry/providers/docs#callouts

Not insisting it should be used here, just pointing out the fact.